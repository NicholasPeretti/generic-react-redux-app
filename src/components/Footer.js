import React from 'react'
import Text from './Text'
import Style from '../style'
import styled from 'styled-components'

const StyledFooter = styled.footer`
  display: flex;
  flex-direction: row;
  justify-content: center;
  height: 80px;
  border-top: 2px solid ${Style.colors.gray};
`

const VericalAlignCenter = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
`

const Footer = () => (
  <StyledFooter>
    <VericalAlignCenter>
      <Text>&#169; AUTO1 Group {new Date().getFullYear()}</Text>
    </VericalAlignCenter>
  </StyledFooter>
)

export default Footer
