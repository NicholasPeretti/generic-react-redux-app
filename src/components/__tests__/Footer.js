import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import Footer from '../Footer'

describe('Footer', () => {
  PresentationalTest(Footer)

  test('Should match snapshot', () => {
    const wrapper = shallow(<Footer/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
