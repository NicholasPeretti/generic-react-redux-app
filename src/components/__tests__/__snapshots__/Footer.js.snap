// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should match snapshot 1`] = `"<footer class=\\"sc-bwzfXH jfoUVV\\"><div class=\\"sc-htpNat dQtbOH\\"><span class=\\"sc-bdVaJa bkmfbd\\">Â© AUTO1 Group 2018</span></div></footer>"`;
